paper-slider.html

```diff
@@ -359,21 +361,29 @@
         this._trackStart(e);
       }
       var x = Math.min(this._maxx, Math.max(this._minx, e.detail.dx));
-      this._x = this._startx + x;
-      this._setImmediateValue(this._calcStep(
-          this._calcKnobPosition(this._x / this._w)) || 0);
       var s =  this.$.sliderKnob.style;
-      s.transform = s.webkitTransform = 'translate3d(' + (this.snaps ?
-          (this._calcRatio(this.immediateValue) * this._w) - this._startx : x) + 'px, 0, 0)';
+      var translateX;
+
+      this._x = this._startx + x;
+      this._setImmediateValue(
+          this._calcStep(this._calcKnobPosition(this._x / this._w)) || 0);
```

Comment:
Usages of `this._calcStep` seem to be followed with `|| 0`. Perhaps `0` should be a built-in default return value for the `_calcStep` method?